### DISCUSSION TITLE
Collapsing Sidebar/Navbar

### DISCUSSION THREAD

**Discussion by FreezeManny** [themesberg/flowbite-react#936] [Q&A]
Hello,
is there an easy way for the sidebar and navbar to colapse on a mobile device, like it does with the regular flowbite? 
Greetings

**Comment by tulup-conner** [comment]
I have traditionally accomplished this by putting the main content of the page - so everything except the Navbar and Sidebar - into a `<div id="..">` element, and then attaching an event listener to clicks inside `<div id="..">`. Then you can set the Navbar and/or Sidebar to collapse when you detect a small screen width.
